bin/runtime/src/lib.rs

```diff
@@ -343,6 +340,7 @@ impl pallet_elections::Config for Runtime {
     type Event = Event;
     type DataProvider = Staking;
     type SessionPeriod = SessionPeriod;
+    type SessionManager = pallet_session::historical::NoteHistoricalRoot<Runtime, Staking>;
```

Comment:
We miss that line in the original change? What's the reason for adding it?

Reply:
this is a way of providing the `SessionManager` into elections pallet